src/Milestones/Milestone.ts

```diff
@@ -0,0 +1,11 @@
+import { IPlayer } from "../PersonObjects/IPlayer";
+
+export class Milestone {
```

Comment:
If we are starting to use functional components it would be good to look into using Typescript types.